extensions/ql-vscode/package.json

```diff
@@ -1732,6 +1732,7 @@
     "format": "prettier --write **/*.{ts,tsx} && eslint . --ext .ts,.tsx --fix",
     "lint": "eslint . --ext .js,.ts,.tsx --max-warnings=0",
     "lint:markdown": "markdownlint-cli2 \"../../**/*.{md,mdx}\" \"!**/node_modules/**\" \"!**/.vscode-test/**\" \"!**/build/cli/v*/**\"",
+    "find-deadcode": "ts-unused-exports find-deadcode-tsconfig.json --showLineNumber --excludePathsFromReport=$(find src test -type f '(' -name jest.config.ts -or -name 'index.tsx' -or -name 'index.ts' ')' -print0 | tr '\\0' ';')'gulpfile.ts/;src/stories/;test/vscode-tests/jest-runner-installed-extensions.ts'",
```

Comment:
Would it make sense to use a script to run this so it's also compatible with Windows and a little bit more readable? It seems like [you can use `ts-unused-exports` as a library](https://github.com/pzavolinsky/ts-unused-exports#usage-as-a-library---from-typescript-was-built-with-es5--commonjs).